prime-router/src/test/kotlin/fhirengine/engine/FhirConverterTests.kt

```diff
@@ -223,4 +242,14 @@ class FhirConverterTests {
         val result = engine.getContentFromFHIR(message, actionLogger)
         assertThat(result).isNotEmpty()
     }
+
+    @Test
+    fun `test getTransformerFromSchema`() {
+        val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.process) as FHIRConverter)
+
+        assertThat(engine.getTransformerFromSchema("")).isNull()
+
+        // Expect a failure since the schema doesn't exist, but this verifies that we tried
+        assertThat { engine.getTransformerFromSchema("some-schema") }.isFailure()
```

Comment:
As for the positive case, this test case was meant to just exercise `getTransformerFromSchema`, which in one case returns a clean null, and in the other case calls a constructor for `FhirTransformer` (which leads to the exception). I don't know that a non-failing constructor call needs to be covered here, since all I wanted to verify here is that the constructor does get called. I'll think about this though, I suppose if we tested that on a schema file that does exist, we could avoid the issue of the `.isFailure` specificity altogether.